.travis.yml

```diff
@@ -28,9 +28,8 @@ before_install:
 
 install:
     - python -m pip install --upgrade pip
-    - python -m pip install --upgrade cmake cython matplotlib mpi4py numpy scipy
+    - python -m pip install --upgrade cmake cython matplotlib mpi4py numpy scipy yt
```

Comment:
I think some of these dependencies are not needed anymore, I only had to list them to build `yt` from source...